File: TPAppel/app/src/main/java/com/example/tpappel/MainActivity.java
package com.example.tpappel;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText numTel;
    Button btnDial;
    EditText editRecherche;
    Button search;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Init();
        btnDial.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                DialNumber();
            }
        });
    }


    private void Init(){
        numTel =  findViewById(R.id.NumTel);
        btnDial =  findViewById(R.id.BtnDial);
        editRecherche =  findViewById(R.id.EditRecherche);
        search =  findViewById(R.id.Search);
    }
    private void DialNumber(){
        String numero="tel:"+numTel.getText().toString();
        Intent dialIntent=new Intent(Intent.ACTION_DIAL, Uri.parse(numero));
        startActivity(dialIntent);
    }
}

File: app/src/main/java/com/example/tpfirebasekotlin/MainActivity.kt
package com.example.tpfirebasekotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    lateinit var ajoutLivre: Button
    lateinit var livreDisp: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Init()
        ajoutLivre.setOnClickListener{
            startActivity(Intent(this@MainActivity,AjoutLivre::class.java))
        }
    }

    fun Init(){
        ajoutLivre=findViewById(R.id.AjoutLivre)
        livreDisp=findViewById(R.id.LivreDispo)
    }
}